report/finished/chapters/motion.tex

```diff
@@ -1,4 +1,40 @@
-\section*{Robot arm}
+\section*{Robot Arm Path Planning} 
+
+
+
+If a robot manipulator is required to visit specific discrete points, waypoints, path planning is the problem of deciding how to move between these points. Planning the path is also important to ensure the movements are smooth, meaning that, since we "control" the speed of each joint, the desired positions and velocities should be continuous.
```

Comment:
We don't "control", we actually control, even if it's open loop so you can remove the citations